translator/tests/acceptance/features/bgp.feature

```diff
@@ -1,22 +1,22 @@
 Feature: block with BGP
   Users can block routes via BGP
 
-  Scenario: We can block a v4 IP
-    When we add 192.0.2.4/32 to the block list
-    Then 192.0.2.4 is blocked
-    And 192.0.2.5 is unblocked
+  Scenario Outline: We can block an IP
+    When we add <route> with <asn> and <community> to the block list
+    Then <route> is blocked
+    And <UnblockIP> is unblocked
 
-  Scenario: We can block a v6 IP
-    When we add 2001:DB8:A::/64 to the block list
-    Then 2001:DB8:A:: is blocked
-    And baba:: is unblocked
+    Examples: data
+    | route           |  asn  | community | UnblockIP |
+    | 192.0.2.4/32    | 64500 | 666       | 192.0.2.5 |
+    | 2001:DB8:A::/64 | 64500 | 666       | baba::    |
 
-  Scenario: We can block, then unblock a v4 IP
-    When we add 192.0.2.4/32 to the block list
-    And we delete 192.0.2.4/32 from the block list
-    Then 192.0.2.4 is unblocked
+  Scenario Outline: We can block an IP
+    When we add <route> with <asn> and <community> to the block list
+    And we delete <route> from the block list
+    Then <UnblockIP> is unblocked
 
-  Scenario: We can block, then unblock a v6 IP
-    When we add 2001:DB8:B::/64 to the block list
-    And we delete 2001:DB8:B::/64 from the block list
-    Then 2001:DB8:B:: is unblocked
+    Examples: data
```

Comment:
Same comments as above.

Reply:
added 32 bit ASN for both, but used the private range.